src/RouterLink.js

```diff
@@ -54,4 +55,29 @@ const RouterLink = {
   },
 }
 
+const withHandleClick = function (tagOrComponent, to, activeClass) {
+  return Vue.component('withHandleClick', {
```

Comment:
Why are you doing `Vue.component` here? That's for registering a global component. It looks like it should with just returning the component defintition.